packages/kit/src/runtime/server/page/index.js

```diff
@@ -99,6 +100,28 @@ export async function render_page(event, page, options, manifest, state, resolve
 		// no server data to prerender. As a result, the load functions and rendering
 		// only occur client-side.
 		if (get_option(nodes, 'ssr') === false && !(state.prerendering && should_prerender_data)) {
+			// if SSR is turned off, and the user makes a request through a non-enhanced form, the action that
+			// returns a value is lost because neither the server nor the client will handle it
```

Comment:
```suggestion
			// if SSR is turned off, and the user makes a request through a non-enhanced form,
			// the returned value is lost because neither the server nor the client will handle it
```